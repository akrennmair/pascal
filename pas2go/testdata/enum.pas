program test;

type foo = ( a, b, c );

var x : foo;

begin
	x := b;
	writeln('x = ', x)
end.
