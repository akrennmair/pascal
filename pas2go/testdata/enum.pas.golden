package main

import (
	"github.com/akrennmair/pascal/pas2go/system"
)

var _ = system.Write

// program test
func main() {
	type (
		foo int
	)
	const (
		a foo = 0
		b foo = 1
		c foo = 2
	)

	var (
		x int
	)
	_ = x

	x = b
	system.Writeln("x = ", x)
}
